spanner/spannertest/README.md

```diff
@@ -12,6 +12,19 @@ production Cloud Spanner. They are reasonable for building tools, or writing
 unit or integration tests. Full-scale performance testing or serious workloads
 should use the production Cloud Spanner instead.
 
```

Comment:
Can we maybe add a section here also that says:

```suggestion
You can use the [Spanner Emulator](https://cloud.google.com/spanner/docs/emulator) for end-to-end functional testing.
```

Reply:
Done.